www/common/translations/messages.de.js

```diff
@@ -803,170 +812,175 @@ define(function () {
-    out.features_f_devices_note = "Zugang zu deinem CryptPad überall mit deinem Benutzer";
+
+    out.features_f_anon = "Alle Funktionen für anonyme Benutzer";
+    out.features_f_anon_note = "Mit einer besseren Benutzbarkeit und mehr Kontrolle über deine Dokumente";
+    out.features_f_cryptdrive1 = "Alle Funktionen des CryptDrives";
+    out.features_f_cryptdrive1_note = "Ordner, geteilte Ordner, Vorlagen, Tags";
+    out.features_f_devices = "Deine Dokumente auf allen deinen Geräten";
+    out.features_f_devices_note = "Überall Zugang zu deinem CryptDrive mit deinem Benutzerkonto";
     out.features_f_social = "Soziale Anwendungen";
-    out.features_f_social_note = "Ein Profil Gestalten, ein Profilbild verwenden, mit Konktakte chatten";
+    out.features_f_social_note = "Ein Profil erstellen, ein Profilbild verwenden, mit Kontakten chatten";
     out.features_f_file1 = "Dateien hochladen und teilen";
-    out.features_f_file1_note = "Dateien mit Freunde teilen, oder sie in Dokumenten einbetten";
-    out.features_f_storage1 = "Langfristige Speicherung (50MB)";
-    out.features_f_storage1_note = "Dateien in deinem CryptDrive sind nicht wegen Aktivitätsmangel gelöscht";
+    out.features_f_file1_note = "Dateien mit Freunden teilen oder sie in Dokumenten einbetten";
+    out.features_f_storage1 = "Langfristige Speicherung (50 MB)";
+    out.features_f_storage1_note = "Dateien in deinem CryptDrive werden nicht wegen Inaktivität gelöscht";
     out.features_f_register = "Registrieren (kostenlos)";
-    out.features_f_register_note = "Keine Email oder persönliche Information nötig";
-       
-    out.features_f_reg = "Alle Funktionen eines gemeldeten Benutzer";
-    out.features_f_reg_note = "Du hilfst die Entwicklung von CryptPad";
-    out.features_f_storage2 = "Grössere Speicherungraum";
-    out.features_f_storage2_note = "Zwischen 5GB und 50GB, abhängig vom selektierten Plan";
-    out.features_f_support = "Schnelleres Support";
-    out.features_f_support_note = "Professionnelles Emailsupport, mit dem Team Plan";
+    out.features_f_register_note = "Keine E-Mail-Adresse oder persönliche Informationen notwendig";
+
+    out.features_f_reg = "Alle Funktionen für angemeldete Benutzer";
+    out.features_f_reg_note = "Du unterstützt die Entwicklung von CryptPad";
+    out.features_f_storage2 = "Mehr Speicherplatz";
+    out.features_f_storage2_note = "Zwischen 5 GB und 50 GB, abhängig vom gewählten Tarif";
+    out.features_f_support = "Schnellerer Support";
+    out.features_f_support_note = "Professioneller Support via E-Mail mit dem Team-Tarif";
     out.features_f_supporter = "Werde ein Unterstützer des Datenschutzes";
-    out.features_f_supporter_note = "Hilfe uns beweisen, dass Software mit eingebauten Datenschutz die Normalität sein sollten";
+    out.features_f_supporter_note = "Hilfe uns beim Beweis, dass Software mit eingebauten Datenschutz die Normalität sein sollte";
```

Comment:
```suggestion
    out.features_f_supporter_note = "Hilfe uns zu beweisen dass Software mit eingebauten Datenschutz die Normalität sein sollte";
```

Reply:
Also: shouldn't that be "Hilf" or "Helfe" instead of "Hilfe"

Reply:
It should be "Hilf".

Reply:
Should be "Hilf uns zu beweisen, dass [...]"